include/deal.II/matrix_free/portable_fe_evaluation.h

```diff
@@ -229,6 +229,22 @@ namespace Portable
     DEAL_II_HOST_DEVICE void
     submit_gradient(const gradient_type &grad_in, int q_point);
 
+    /**
+     * Same as above, except that the quadrature point is computed from the
```

Comment:
All these comments have to be approved (not your fault as you just copied comment from above)